doc/connectivity/resting_state_networks.rst

```diff
@@ -25,76 +24,66 @@ Multi-subject ICA: CanICA
       fMRI datasets", `NeuroImage Vol 51 (2010)
       <http://www.sciencedirect.com/science/article/pii/S1053811910001618>`_, p. 288-299
 
-Data preparation: retrieving example data
------------------------------------------
+Objective
+----------
+ICA is a useful approach for finding independent sources of fMRI
+timeseries covariance without needing to define any priors. ICA
```

Comment:
```suggestion
images. ICA
```
Technically, in fMRI, ICA is applied in the spatial domain, and not the temporal domain. Let's stay vague :).